test/addMissingDependencies/coreConfiguration/configuration.js

```diff
@@ -0,0 +1,39 @@
+/*!
+ * ${copyright}
+ */
+
+// A module
+sap.ui.define([],
+	function() {
+		"use strict";
+
+		/**
+		 *
+		 * @type {{}}
+		 */
+		var A = {};
+
+		/**
+		 *
+		 * @param oParam
+		 * @param iIndex
+		 */
+		A.x = function (oParam, iIndex) {
```

Comment:
add some code examples with "getConfiguration()" that shouldn't be replaced

Reply:
done